force-app/main/default/lwc/edaSettingsNavigation/edaSettingsNavigation.js

```diff
@@ -0,0 +1,112 @@
+                                sectionKey: "autodeleterecordtypes",
+                            },
+                            { label: "Administrative Account Naming", sectionKey: "administrativeaccountnaming" },
+                            { label: "Household Account Naming", sectionKey: "householdAccountNaming" },
+                            { label: "Rename Household Accounts", sectionKey: "renamehouseholdaccounts" },
+                        ],
+                        isActive: true,
+                    },
+                    {
+                        label: "Contact Information",
+                        page: "contactinfosettings",
+                        sectionkey: "contactinfosettings",
+                        menuItems: [
+                            { label: "Default Contact Language Fluency", sectionKey: "defaultcontactlanguagefluency" },
+                            { label: "Require Preferred Email", sectionKey: "requirepreferredemail" },
+                            { label: "Enhanced Preferred Phone Functionality", sectionKey: "enhancedpreferredphone" },
+                            { label: "Default Preferred Phone", sectionKey: "defaultpreferredphone" },
+                        ],
+                    },
+                    {
+                        label: "Addresses",
+                        page: "addresssettings",
+                        sectionkey: "addresssettings",
+                        menuItems: [
+                            { label: "Allow Multiple Contact Addresses", sectionKey: "contactmultipleaddresses" },
+                            {
+                                label: "Enable Multiple Addresses for Account Types",
+                                sectionKey: "addressaccountrecordtypes",
+                            },
+                            {
+                                label: "Update Address Records for Simple Address Changes",
+                                sectionKey: "simpleaddresschangeisupdate",
+                            },
+                        ],
+                    },
+                ],
+            },
+            {
+                label: "Affiliations",
+                page: "affiliations",
+                sectionkey: "affiliations",
+                menuItems: [{ label: "Enforce Record Type Validation", sectionKey: "enforcerecordtypevalidation" }],
+            },
+            {
+                label: "Courses and Enrollments",
+                sectionkey: "coursesandenrollments",
+            },
+            {
+                label: "System Tools",
+                sectionkey: "systemtools",
+            },
+        ],
+    };
+
+    @api setActivePage(pageName) {
+        this.settingsNavigationViewModel.navigationSections.forEach((navigationSection) => {
+            if (navigationSection.page === pageName) {
+                navigationSection.isActive = true;
+            } else {
+                navigationSection.isActive = undefined;
```

Comment:
Just curious as to why we opted to set to undefined here instead of false?

Reply:
This is following truthy/falsy. More importantly, this is a format of defining variables that can be important when dealing with some attributes in lwc, such as disabled on input affordances. In a 'fun' experience with said attributes:

- When a disabled attribute is undefined, the component is considered to not be disabled.
- When disabled === false, the component is disabled. This was the headscratcher moment for me.
- When disabled === true, the component is disabled.
- Thus, when disabled is defined, the component is considered disabled. Fun with Javascript!